x11-misc/fbpanel/fbpanel-7.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit toolchain-funcs
+
+DESCRIPTION="light-weight X11 desktop panel"
+HOMEPAGE="https://aanatoly.github.io/fbpanel/"
+SRC_URI="https://github.com/aanatoly/fbpanel/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~mips ~ppc ~ppc64 ~x86"
+IUSE="alsa"
+
+RDEPEND="
+	dev-libs/glib:2
+	alsa? ( media-libs/alsa-lib )
+	x11-libs/gdk-pixbuf:2[X]
+	x11-libs/gtk+:2
+	x11-libs/libX11
+"
+DEPEND="
+	${RDEPEND}
+	dev-lang/python:2.7
```

Comment:
python cannot be used like this. If python is a build-time dep -> `python-any-r1`, if its also a runtime time -> `python-single-r1`.